frontend/packages/metal3-plugin/src/components/machine-cell.tsx

```diff
@@ -1,21 +1,18 @@
 import * as React from 'react';
-import { Link } from 'react-router-dom';
-import { AddCircleOIcon } from '@patternfly/react-icons';
 
-import { DASH, StatusIconAndText } from '@console/shared';
+import { DASH } from '@console/shared';
 import { MachineModel } from '@console/internal/models';
-import { ResourceLink, RequireCreatePermission } from '@console/internal/components/utils';
+import { ResourceLink } from '@console/internal/components/utils';
 import { referenceForModel, K8sResourceKind } from '@console/internal/module/k8s';
 
 import { getHostMachineName } from '../selectors';
-import { canHostAddMachine, HostMultiStatus } from '../utils/host-status';
 
 interface MachineCellProps {
   host: K8sResourceKind;
-  status: HostMultiStatus;
+  HostMultiStatus;
 }
 
-const MachineCell: React.FC<MachineCellProps> = ({ host, status }) => {
+const MachineCell: React.FC<MachineCellProps> = ({ host }) => {
   const machineName = getHostMachineName(host);
 
   const {
```

Comment:
Please replace this metadata definition with:
```suggestion
  const namespace = getNamespace(host)
```